src/ReinforcementLearningZoo/src/algorithms/policy_gradient/ppo.jl

```diff
@@ -1,4 +1,5 @@
 export PPOPolicy, PPOTrajectory, MaskedPPOTrajectory
+import ReinforcementLearningCore.normlogpdf
```

Comment:
Should we make this consistent with the previous example and do:
```suggestion
import ReinforcementLearningCore: normlogpdf
```